examples/fabric-bridge-app/linux/RpcClient.cpp

```diff
@@ -0,0 +1,169 @@
+/*
+ *
+ *    Copyright (c) 2024 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "RpcClient.h"
+
+#include <string>
+#include <thread>
+#include <unistd.h>
+
+#include "fabric_admin_service/fabric_admin_service.rpc.pb.h"
+#include "pw_assert/check.h"
+#include "pw_hdlc/decoder.h"
+#include "pw_hdlc/default_addresses.h"
+#include "pw_hdlc/rpc_channel.h"
+#include "pw_rpc/client.h"
+#include "pw_stream/socket_stream.h"
+
+namespace {
+
+constexpr size_t kMaxTransmissionUnit = 256;
+constexpr uint32_t kRpcTimeoutMs      = 1000;
+const char * rpcServerAddress         = "127.0.0.1";
+
+pw::stream::SocketStream rpcSocketStream;
+
+// Set up the output channel for the pw_rpc client to use.
+pw::hdlc::RpcChannelOutput hdlc_channel_output(rpcSocketStream, pw::hdlc::kDefaultRpcAddress, "HDLC channel");
+
+// An array of RPC channels (channels) is created, each associated with an HDLC channel output.
+// This sets up the communication channels for RPC calls.
+pw::rpc::Channel channels[] = { pw::rpc::Channel::Create<1>(&hdlc_channel_output) };
+
+// Initialize the RPC client with the channels.
+pw::rpc::Client client(channels);
+
+// Generated clients are namespaced with their proto library.
+using FabricAdminClient = chip::rpc::pw_rpc::nanopb::FabricAdmin::Client;
+
+// RPC channel ID on which to make client calls. RPC calls cannot be made on
+// channel 0 (Channel::kUnassignedChannelId).
+constexpr uint32_t kDefaultChannelId = 1;
+
+// Function to process incoming packets
+void ProcessPackets()
```

Comment:
This is an exact duplicate of the other file (examples/fabric-admin/rpc/RpcClient.cpp). Please factor the code into a single reusable module.